packages/chart/src/FigureChartModel.js

```diff
@@ -566,6 +566,13 @@ class FigureChartModel extends ChartModel {
       if (yLow && yHigh && yLow !== y) {
         seriesData.error_y = ChartUtils.getPlotlyErrorBars(y, yLow, yHigh);
       }
+    } else if (plotStyle === dh.plot.SeriesPlotStyle.TREEMAP) {
+      const { ids, labels } = seriesData;
+      if (ids && !labels) {
```

Comment:
What's the type of `labels`? If it's an array, is it enough to only check for truthy/falsy value here?

Reply:
Yes, it's an array. You're right I should be checking for undefined instead.

Reply:
Actually, I was thinking about the situation when the `labels` array is truthy but empty. Is it possible we get an empty array from the API?

Reply:
Right - if we get an empty array, we should be keeping that. It's _possible_, but would also mean there's nothing to plot. Basically means you created a plot with an empty data set.